src/util/helpers.ts

```diff
@@ -28,6 +29,15 @@ export const formatDec = (num: number) => {
   return Number(num.toFixed(Config.DECIMAL)).toString();
 };
 
+export const toBitcoinAmount = (amount: number | string) => {
+  const txAmount = typeof amount !== 'string' ? amount.toString() : amount;
+  const value = toBitcoin(txAmount);
+  return Number(value.toFixed(Config.DECIMAL)).toString();
+};
+
+export const getUsdValue = (amount: number, tokelPriceUSD: number) =>
```

Comment:
I'd love if `amount` (in the function above as well) could be named `amountInSatoshis` as to make really explicit what we're handling.